packages/site-admin/.storybook/preview.tsx

```diff
@@ -0,0 +1,36 @@
+import { RegistryProvider } from '@wordpress/data';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { createMockRegistry } from './mock-registry';
+import stores from './mock-registry/stores';
+
+const siteKeys = Object.keys( stores );
+
+const withRegistryProvider = ( Story, context ) => {
+	const { site } = context.globals;
+	const registry = createMockRegistry( stores[ site ] );
+
+	return (
+		<RegistryProvider value={ registry }>
+			<Story />
+		</RegistryProvider>
+	);
+};
+
+export const globalTypes = {
+	site: {
+		name: 'Site',
+		description: 'Site for testing',
+		defaultValue: siteKeys[ 0 ],
+		toolbar: {
+			icon: 'globe',
```

Comment:
Nit: It might be better to pick a different icon here. The reason is that we'll probably add an RTL toggle button at some point, and that will likely want to use the `globe` icon (for consistency with the WP Storybook).

Reply:
Ive used `home`

Reply:
<img width="354" alt="image" src="https://github.com/user-attachments/assets/1b88625a-7432-493b-a506-d6aa31f09834" />